test/minikube/minikube_rolling_upgrade_test.go

```diff
@@ -140,6 +142,9 @@ func TestKubernetesUpgradeFullDatabaseMinorVersion(t *testing.T) {
 
 		helm.Upgrade(t, &databaseOptions, testlib.DATABASE_HELM_CHART_PATH, databaseHelmChartReleaseName)
 
+		// make sure that we only have 1 TE and not 2
+		testlib.SetDeploymentUpgradeStrategyToRecreate(t, namespaceName, fmt.Sprintf("te-%s-nuodb-cluster0-demo", databaseHelmChartReleaseName))
```

Comment:
I can't map the comment with the upgrade strategy change. Is that so we need to change the strategy before upgrading when we have only one TE replica?